app/api/users.py

```diff
@@ -20,15 +21,18 @@
 from app.models.feedback import Feedback
 from app.models.speaker import Speaker
 from app.models.session import Session
-from app.api.helpers.exceptions import ConflictException
-from app.api.helpers.db import safe_query
+from app.api.helpers.exceptions import ConflictException, UnprocessableEntity
+from app.api.helpers.db import safe_query, get_count
 from app.models.user import User
 from app.models.users_events_role import UsersEventsRoles
 from app.models.ticket_holder import TicketHolder
 from app.api.helpers.exceptions import ForbiddenException
 from app.api.helpers.permission_manager import has_access
 
 
+user_misc_routes = Blueprint('user_misc', __name__, url_prefix='/v1')
```

Comment:
We used to do this in v1. Please check ```app/api/__init__.py``` file. We declare endpoints in that file now.